templates/repo/projects/view.tmpl

```diff
@@ -34,8 +34,39 @@
 			</div>
 		</div>
 		<div class="ui divider"></div>
+		<div class="ui two column stackable grid">
+			<div class="column">
+				<h2 class="project-title">{{$.Project.Title}}</h2>
+				<div class="content project-description">{{$.Project.RenderedContent|Str2html}}</div>
+			</div>
+			{{if and (or $.CanWriteIssues $.CanWritePulls) (not $.Repository.IsArchived)}}
+			<div class="column right aligned">
+				<div class="ui compact right small menu">
+					<a class="item" href="{{$.RepoLink}}/projects/{{.Project.ID}}/edit" data-id={{$.Project.ID}} data-title={{$.Project.Title}}>
+						{{svg "octicon-pencil"}}
+						<span class="mx-3">{{$.i18n.Tr "repo.issues.label_edit"}}</span>
+					</a>
+					{{if .Project.IsClosed}}
+						<a class="item link-action" href data-url="{{$.RepoLink}}/projects/{{.Project.ID}}/open">
+							{{svg "octicon-check"}}
+							<span class="mx-3">{{$.i18n.Tr "repo.projects.open"}}</span>
+						</a>
+					{{else}}
+						<a class="item link-action" href data-url="{{$.RepoLink}}/projects/{{.Project.ID}}/close">
+							{{svg "octicon-x"}}
```

Comment:
```suggestion
							{{svg "octicon-skip"}}
```

May be better fitting

Reply:
I used the same icon as `projects/list.tmpl`. I think if we change it here, it should be changed there too.

Reply:
Yeah please change it there too.